includes/RestApi/FlowController.php

```diff
@@ -181,7 +161,7 @@ public function save_onboarding_flow_data( \WP_REST_Request $request ) {
 		}
 
 		// save data to database
-		if ( ! $this->update_wp_options_data_in_database( $flow_data ) ) {
+		if ( ! \update_option( Options::get_option_name( 'flow' ), $flow_data ) ) {
```

Comment:
Why are we not using a method for this from FlowService? considering we have one for `FlowService::read_flow_data_from_wp_option();`

Cause I feel I might miss something while doing copy-paste.